src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1375,7 +1375,14 @@ private void visitFunctionParameters(FunctionType ftype, boolean emitTemplatized
         if (names != null && names.hasNext()) {
           emitNoSpace(names.next());
         } else {
-          emitNoSpace("" + pName++);
+          String pName;
+          if (paramCount < 26) {
```

Comment:
I don't understand why you bother with this and not just generate p1, p2, p3, p4, and so on? What's so awesome about 'a', 'b', 'c' in comparison to that? It complicates the code substantially over just `String pName = "p" + (++paramCount);`